daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/dpath/NodeInfo.scala

```diff
@@ -534,7 +534,9 @@ object NodeInfo extends Enum {
     }
 
     trait PrimNumeric { self: Numeric.Kind =>
-      def width: MaybeInt
+      val isSigned: Boolean
```

Comment:
Make this a def. Def's can be overridden with a val, but not the other way around, so def is a bit more flexible for implementations.